modules/passkey/src/types/solc.d.ts

```diff
@@ -0,0 +1,41 @@
+declare module 'solc' {
+  export interface CompilerInput {
+    language: string
+    sources: {
+      [fileName: string]: {
+        content: string
+      }
+    }
+    settings: {
+      outputSelection: {
+        [fileName: string]: {
+          [contractName: string]: string[]
+        }
+      }
+    }
+  }
+
+  export interface CompilerOutput {
+    contracts: {
+      [fileName: string]: {
+        [contractName: string]: {
+          abi: any[]
+          evm: {
+            bytecode: {
+              object: string
+            }
+          }
+        }
+      }
+    }
+    sources: {
+      [fileName: string]: {
+        ast: any
+      }
+    }
+  }
+
+  export function compile(input: string): string
+
+  export function loadRemoteVersion(version: string, callback: (err: Error, solc: any) => void): void
```

Comment:
nit:lints

Reply:
What do you mean by lints? I see no linting errors